src/gatherers/accessibility.js

```diff
@@ -42,49 +42,49 @@ class Accessibility extends Gather {
 
   static _errorAccessibility(errorString) {
     return {
-      manifest: {
+      accessibility: {
         raw: undefined,
         value: undefined,
         debugString: errorString
       }
     };
   }
 
-  afterPageLoad(options) {
-    const driver = options.driver;
-
-    const fetchResults = count => {
-      if (count === 0) {
-        return Promise.resolve({
-          result: {
-            value: null
+  fetchResults(driver, count) {
+    if (count === 0) {
+      return Promise.resolve({
+        result: {
+          value: null
+        }
+      });
+    }
+
+    return new Promise((resolve, reject) => {
+      setTimeout(_ => {
```

Comment:
Just because I didn't want to thrash the check for the axe results. However if #247 lands this code will vanish entirely.... just got to find out what the issue is there :-/